src/bin/cli.js

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env node
+const { program } = require("commander");
+const package = require("../../package.json");
```

Comment:
Rename "package" identifier to prevent potential conflicts with future evolutions of the JavaScript language.

Reply:
I see. You are talking about the reserved keyword. OK.

Reply:
Done.